README.md

```diff
@@ -101,172 +138,110 @@ the PuppetLint configuration by defining the task yourself.
+* To disable a check, add the following line after the `require` statement in your `Rakefile`:
 
-At the moment, the following tests have been implemented:
+  ``` ruby
+  PuppetLint.configuration.send("disable_<check name>")
+  ```
 
-### Spacing, Indentation & Whitespace
+  For example, to disable the 140-character check, add:
 
- * Must use two-space soft tabs.
- * Must not use literal tab characters.
- * Must not contain trailing white space.
- * Should not exceed an 140 character line width
-   * An exception has been made for `source => 'puppet://...'` lines as
-     splitting these over multiple lines decreases the readability of the
-     manifests.
- * Should align arrows (`=>`) within blocks of attributes.
+  ``` ruby
+  PuppetLint.configuration.send("disable_140chars")
+ ```
 
-### Quoting
+* To set the Lint Rake task to ignore certain paths:
 
- * All strings that do not contain variables should be enclosed in single
-   quotes.
-   * An exception has been made for double quoted strings containing \n or \t.
- * All strings that contain variables must be enclosed in double quotes.
- * All variables should be enclosed in braces when interpolated in a string.
- * Variables standing by themselves should not be quoted.
+  ``` ruby
+  PuppetLint.configuration.ignore_paths = ["vendor/**/*.pp"]
+  ```
 
-### Capitalization
- * All variables should be in lowercase
+* To set a pattern of files that Lint should check:
 
-### Resources
+  ``` ruby
+  # Defaults to `**/*.pp`
+  PuppetLint.configuration.pattern = "modules"
+  ```
 
- * All resource titles should be quoted.
-   * An exception has been made for resource titles that consist of only
-     a variable standing by itself.
- * If a resource declaration includes an `ensure` attribute, it should be the
-   first attribute specified.
- * Symbolic links should be declared by using an ensure value of `link` and
-   explicitly specifying a value for the `target` attribute.
- * File modes should be represented as a 4 digit string enclosed in single
-   quotes or use symbolic file modes.
+## Options
 
-### Conditionals
-
- * You should not intermingle conditionals inside resource declarations (i.e.
-   selectors inside resources).
- * Case statements should have a default case.
-
-### Classes
+See `puppet-lint --help` for a full list of command line options and checks.
 
- * Relationship declarations with the chaining syntax should only be used in
-   the 'left to right' direction.
- * Classes should not be defined inside a class.
- * Defines should not be defined inside a class.
- * Classes should not inherit between namespaces.
- * Required parameters in class & defined type definitions should be listed
-   before optional parameters.
- * When using top-scope variables, including facts, Puppet modules should
-   explicitly specify the empty namespace.
- * Chaining operators should appear on the same line as the right hand operand.
-
-## Disabling checks
-
-### puppet-lint
+## Implemented Tests
```

Comment:
They should be called Checks, and it might be better to link to their page on the github pages site than duplicate that info.